### DISCUSSION TITLE
Adding meta <link> in the document head

### DISCUSSION THREAD

**Discussion by tigawanna** [rakkasjs/rakkasjs#159] [Q&A] [answered]
Tried  to add favico  files with the new <Head/> tag , it doesn't seem to have a prop for it an doesn't work if I pass in the meta tags wrapped in a string as the children , tried this too 

```ts
      emitToDocumentHead() {
        // Return a string or ReactElement to emit
        // some HTML into the document's head.
        const cookie_theme = requestContext?.cookie?.theme;
        // inject a script into the page's head to set the data-theme attribute before tha
        // page loads to avoid a flash of the old theme
        return `
      <link rel="icon" type="image/x-icon" href="/favicon.ico" />
      <script>
      (function() {
        document.documentElement.setAttribute("data-theme", "${cookie_theme}");
      })();
     </script>
     `;
      },

```

**Comment by CanRau** [comment, marked as answer]
Not sure you saw my message on Discord, just stumbled upon the discussion here

```tsx
<Head
  title="Hello Rakkas"
  faviconIco={{
    tagName: "link",
    rel: "icon",
    href: "/favicon.ico",
  }}
  {..etc}
/>
```

this works great for me, you can basically anything in there, by the way you can also call the prop `favicon` instead of `faviconIco` or anything you like, and add multiple using differently named props

**Reply by tigawanna** [reply]
Thanks, this works, Am sorry I don't hang around Discord that much because of the spammy notifications, 
I'll try and figure out how to allow notifications  from the Rakkas server 

i wasusing this sa work around
```tsx
      emitToDocumentHead() {
        // Return a string or ReactElement to emit
        // some HTML into the document's head.
        const cookie_theme = requestContext?.cookie?.theme;
        // inject a script into the page's head to set the data-theme attribute before tha
        // page loads to avoid a flash of the old theme
        return `
    <link rel="icon" type="image/svg+xml" href="/site.svg" />
   <script>
      (function() {
        document.documentElement.setAttribute("data-theme", "${cookie_theme}");
      })();
     </script>
     `;
      },
```

Yours is better since mine wasn't working with .icos

**Reply by CanRau** [reply]
Glad it works now 🙌

I disable notifications on all servers but a few and even in those few I don't disable notifications entirely I usually mute channels, this helps a lot. The rest I have to manually check if something happened 🤷🏻‍♂️
